src/test/java/com/fasterxml/jackson/databind/objectid/TestObjectIdDeserialization.java

```diff
@@ -466,4 +486,60 @@ public void testNullObjectId() throws Exception
         assertNotNull(value);
         assertEquals(3, value.value);
     }
+
+    /*
+    /*****************************************************
+    /* Unit tests in conjunction with DeserializationFeature.FAIL_ON_UNRESOLVED_OBJECT_IDS
+    /*****************************************************
+     */
+
+
+    private final ObjectMapper DEFAULT_MAPPER = newJsonMapper();
+
+    private final ObjectMapper DISABLED_MAPPER = newJsonMapper()
+        .configure(DeserializationFeature.FAIL_ON_UNRESOLVED_OBJECT_IDS, false);
```

Comment:
Going forward let's try to use `JsonMapper.builder()` so code will merge and work with 3.0. I will change this after merging.